source/slang/hlsl.meta.slang

```diff
@@ -13298,8 +13310,8 @@ ${
 // further clutter the original type declarations.
 }
 
-__generic<T, Shape: __ITextureShape, let sampleCount:int, let isShadow:int, let format:int>
-extension __TextureImpl<T,Shape,0,0,sampleCount,0,isShadow,0,format>
+__generic<T, Shape: __ITextureShapeTraditional, let sampleCount:int, let isShadow:int, let format:int>
```

Comment:
Looking here I think we can keep our existing TextureImpl unchanged. The :__ITextureShapeTraditional constraint will already exclude the Subpass shape.